packages/os/bootstrap-commands.service

```
@@ -0,0 +1,17 @@
+[Unit]
+Description=Bootstrap Commands
+After=settings-applier.service apiserver.service
+Requires=settings-applier.service apiserver.service
+Before=preconfigured.target
+RefuseManualStart=true
+RefuseManualStop=true
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/bootstrap-commands
+RemainAfterExit=true
+StandardError=journal+console
+SyslogIdentifier=bootstrap-commands
+
+[Install]
+WantedBy=preconfigured.target
```

This needs to be `RequiredBy` or else failures will not stop the system from reaching `preconfigured.target`.
```suggestion
[Install]
RequiredBy=preconfigured.target
```